README.md

```diff
@@ -136,12 +136,18 @@ You can omit point name to automatically use the current directory's name instea
 wd foo
 ```
 
-* You can also warp to a directory within `foo`, with autocompletion:
+* You can also warp to a directory within `foo`:
```

Comment:
Currently, doing `wd foo b<TAB>` will autocomplete to `wd foo bar`. 
Are we proposing to change this with this PR? As far as I can tell, this PR doesn't prevent that behavior, though I believe it won't work with the new slash behavior. 
Am I correct?

Reply:
I could replicate #51 so I've removed the autocompletion guarantee until it's bug-free. Technically not too related to this specific PR, could just be put straight onto master